src/js/components/spinner.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import classnames from "classnames";
+
+const BASE_CLASS = "mw-spinner";
+
+export default class Spinner extends React.Component {
+
+	render() {
+		const classes = [
+			BASE_CLASS,
+			this.props.className
+		];
+
+		const style = { backgroundColor: this.props.color };
```

Comment:
Since this is set by STATUS_COLOR which never changes, would it make more sense to not do any CSS in the JS and move this all to the SCSS files?